### DISCUSSION TITLE
有没有修改血量内力等数值上限的办法？

### DISCUSSION THREAD

**Discussion by alexpanda1** [jynew/jynew#1024] [Ideas]
不懂编程，纯好奇。

**Comment by qige2016** [comment]
https://github.com/jynew/jynew/blob/6999349e2b6ac497b978c6ed1398ccc9c206a2f4/jyx2/Assets/Scripts/LuaCore/Jyx2LuaBridge.cs#L672
https://github.com/jynew/jynew/blob/6999349e2b6ac497b978c6ed1398ccc9c206a2f4/jyx2/Assets/Scripts/LuaCore/Jyx2LuaBridge.cs#L643

**Reply by alexpanda1** [reply]
赞美你~
